aztec/src/main/kotlin/org/wordpress/aztec/AztecTagHandler.kt

```diff
@@ -93,7 +98,12 @@ class AztecTagHandler(val context: Context, val plugins: List<IAztecPlugin> = Ar
                 return true
             }
             LIST_UL -> {
-                handleElement(output, opening, createUnorderedListSpan(nestingLevel, alignmentRendering, AztecAttributes(attributes)))
+                val lastSpan = tagStack.lastOrNull()
+                if (!opening && lastSpan is AztecTaskListSpan) {
```

Comment:
If we're closing the tag and the last span is `AztecTaskListSpan`, we know that the list contains a task list so we have to close the correct span.

Reply:
I've implemented your solution. I've still used `!opening && lastSpan is AztecTaskListSpan` as the alternative because I think that otherwise it might affect a nested list. I'm not sure though.